plugins/intel-spi/fu-plugin-intel-spi.c

```diff
@@ -26,4 +28,10 @@ fu_plugin_init (FuPlugin *plugin)
 	fu_plugin_add_firmware_gtype (plugin, NULL, FU_TYPE_EFI_FIRMWARE_VOLUME);
 	fu_plugin_add_firmware_gtype (plugin, NULL, FU_TYPE_IFD_BIOS);
 	fu_plugin_add_firmware_gtype (plugin, NULL, FU_TYPE_IFD_FIRMWARE);
+	fu_plugin_add_udev_subsystem (plugin, "pci");
```

Comment:
at least for now, I would think you want to have a check if lockdown is set to only initiate all these attributes if lockdown _isn't_ set.

Reply:
Added, good idea.